packages/plugins/plugin-wnfs/tsconfig.json

```diff
@@ -1,83 +1,69 @@
 {
-  "extends": [
-    "../../../tsconfig.base.json"
-  ],
+  "extends": ["../../../tsconfig.base.json"],
   "compilerOptions": {
-    "lib": [
-      "DOM",
-      "ESNext"
-    ],
-    "outDir": "dist/types",
-    "types": [
-      "node"
-    ]
+    "lib": ["DOM", "ESNext"],
```

Comment:
I've noticed this removed in lots of packages, but not others, is there a reason it's needed in some places or could it be removed from them as well?